### DISCUSSION TITLE
react-native-template-typescript updates

### DISCUSSION THREAD

**Discussion by radko93** [reactwg/react-native-new-architecture#22] [Libraries]
We have updated the official template with RN 0.68.0-rc.4 and everyone can try that by using 6.10.0-rc.1 or `next` tag (versions are not tied to RN versions as we tend to have more updates for the template). We will update the template when the stable release is out. 

To try the latest pre-release version:
```sh
npx react-native init MyApp --template react-native-template-typescript@next
```

This template only adds TS tooling to `react-native init`, native part of the template is identical. Suggestions or comments regarding new architecture and/or codegen are welcome.

**Comment by cortinico** [comment]
Thanks for the update @radko93 🙏 Have you noticed any problem with missing types (here https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-native) for the New Architecture?
